usr/lib/common/object.c

```diff
@@ -444,6 +444,47 @@ CK_BBOOL object_is_session_object(OBJECT * obj)
         return TRUE;
 }
 
+// object_is_extractable()
+//
+CK_BBOOL object_is_extractable(OBJECT * obj)
```

Comment:
Isn't extractable a property of a key?  Why do we have it on object level?

Reply:
Well, could be moved to key.c as key_object_is_extractable() , but since we don't yet have any other such key-object helpers I am OK with either way.

Reply:
ok, let's leave this for now.